locales.json

```diff
@@ -11,6 +11,14 @@
     "label": "Español",
     "lang": "es"
   },
+  "it": {
+    "label": "Italiano",
+    "lang": "it"
+  },
+  "ko": {
+    "label": "한국어",
+    "lang": "ko"
+  },
```

Comment:
those _ko and _it only exists because they were in v1 docs, but I don't think they should be enabled, there's next to zero content

Reply:
Ok I guess we'll remove it completely then for now